changelogs/fragments/59068-fix doc for yum_repository.py

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- fix doc for yum_repository.py (https://github.com/ansible/ansible/pull/59068)
```

Comment:
```suggestion
- Fix doc for proxy_username and proxy_password in yum_repository.py (https://github.com/ansible/ansible/pull/59068).
```

Change extension of file to `yml` from `py`